wallet/lib/services/data_stores/remote_data_store.dart

```diff
@@ -126,6 +126,12 @@ abstract class RemoteDataStore {
   /// else will throw error
   Future<List<NftOwnershipHistory>> getNftOwnershipHistory({required String itemId, required String cookBookId});
 
+  /// This method is used to get history of nft owners
+  /// Input: [cookBookID] and [recipeId] of the NFT
```

Comment:
These fields should match the actual params. `cookBookID` should be `cookbookId`